### DISCUSSION TITLE
Activating `citar-open-entry` from search menu

### DISCUSSION THREAD

**Discussion by pprevos** [emacs-citar/citar#838] [Ideas]
Hi Bruce,

I was looking at the code to try and add an option to jump to the relevant source (`citar-open-entry`) file directly from the Citar completion menu. So an additional option to files, links and notes could be source. I currently use Embark to access this function.

I find the code a bit hard to reverse-engineer. Any suggestion how to achieve this?

P:)

**Comment by bdarcus** [comment]
> I was looking at the code to try and add an option to jump to the relevant source (`citar-open-entry`) file directly from the Citar completion menu. So an additional option to files, links and notes could be source.

You mean to add to `citar-open`?

If yes, am not sure sure, but had the impression here it would require some refactoring on our end.

cc @roshanshariff

**Comment by pprevos** [comment]
If it is a big job then perhaps it is not worth prioritising for a minor convenience.
